docs/going-into-production.rst

```diff
@@ -0,0 +1,175 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-auto-boot:
+
+Manual bootstrapping
+====================
+
+:ref:`Single host auto-bootstrapping <auto-bootstrapping>` is useful for
+development environments. However, auto-bootstrapping is typically too slow for
```

Comment:
Feels to me a bit as if this is jumping in too quickly. Is there enough context established at this point so that readers know about the bootstrapping process ?

I'm also not sure about the argument that

> auto-bootstrapping is typically too slow

I think the speed of the bootstrapping process itself can be neglected. The primary reason is to ensure that in production you've the right configuration (multiple nodes that are eligible as master nodes, and that all are part of the  election process)

Reply:
okay expanded this section a lot. will commit a fixup in a min